LuaMenu/widgets/api_download_handler.lua

```diff
@@ -152,6 +152,11 @@ local function AssociatedSpringDownloadID(springDownloadID, name, fileType)
 end
 
 local function RemoveDownload(name, fileType, putInRemoveList, removalType)
+	-- The changes to this func deserve some documentation:
+	-- Often, a map download wont truly fail, it will download just fine but return that it failed,
+	-- and retrying it up to retrycount times will succeed
+	-- a game download will also return with failure often, but a vfs.scanalldirs()
+	-- actually finds it, so we need to check for that to decide whether to
```

Comment:
whether to? .. retry?
seems like you stopped mid-sentence

also some spelling:
"wont" -> won't

will succeed
a game download ->
will succeed.
A game download